src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class Circle extends Figure {
+    private double radius;
+    private double area;
```

Comment:
Think about which variables should be local in the method and which should be class-level. The 'area' variable seems to be used only in the context of 'calculateArea()' and 'draw()' methods, so it could be a local variable inside these methods instead of a class field.